indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3648,9 +3715,10 @@ private void updateCurrentOffsets() throws InterruptedException, ExecutionExcept
 
   protected Map<PartitionIdType, SequenceOffsetType> getHighestCurrentOffsets()
   {
+    Map<PartitionIdType, SequenceOffsetType> offsetsFromMetadataStorage = getOffsetsFromMetadataStorage();
```

Comment:
Please utilize this map instead of calling `getOffsetsFromMetadataStorage` subsequently in the code